atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -143,11 +143,9 @@ def set_help(self, extra, worker_metadatas):
                 extra['image']['help'] = None
 
     def set_media_types(self, extra, worker_metadatas):
```

Comment:
shouldn't the logic be 
`if pulp_pull_results and pulp_pull_results[1]:
      extra['image']['media_types'] = json.loads(pulp_pull_results[1])
      return`

and then the original logic otherwise? osbs-4285 says "otherwise, look for a media-types annotation on the worker build"